cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -1628,3 +1628,24 @@ roleRef:
   name: rbd-external-provisioner-runner
   apiGroup: rbac.authorization.k8s.io
 # OLM: END CSI RBD CLUSTER ROLEBINDING
+---
```

Comment:
A couple final small things:
- Could you move this new CRD up higher in the yaml next to the other CRDs? 
- There are some `# OLM` comments also to put around the CRD